content/intro-to-storybook/react/en/creating-addons.md

```diff
@@ -5,9 +5,9 @@ description: 'Learn how to build your own addons that will super charge your dev
 commit: 'bebba5d'
 ---
 
-In the previous chapter we were introduced to one of the key features of Storybook, its robust system of [addons](https://storybook.js.org/addons/introduction/), which can be used to enhance not only yours but also your team's developer experience and workflows.
+Throughout this tutorial we were introduced to one of the key features of Storybook, its robust system of [addons](https://storybook.js.org/addons/introduction/), which can be used to enhance not only yours but also your team's developer experience and workflows.
```

Comment:
```suggestion
Earlier, we introduced a key Storybook feature: the robust [addons](https://storybook.js.org/addons/introduction/) ecosystem. Addons are used to enhance your developer experience and workflows.
```

Reply:
Changed with the latest commit. Marking it as solved